src/docs/economy.mdx

```diff
@@ -10,24 +10,24 @@ custom_edit_url: https://github.com/EarthMC/earthmc-docs/blob/main/src/docs/econ
 The currency on EarthMC is based on [gold ingots](https://minecraft.fandom.com/wiki/Gold_Ingot). 1 gold ingot is 1 unit of currency. The EarthMC economy is inflation resistant with efficient [gold sinks](https://en.wikipedia.org/wiki/Gold_sink). Gold is always in high demand which in turn creates a great gameplay experience and market environment. The large majority of items and services sold are provided by the players.
 
 ## Store your gold
-The safest place for gold is in an ender chest. Gold can also be deposited in to the virtual town/nation bank. This will allow residents with appropriate permissions to withdraw gold and use it for town/nation expenses such as claiming. To deposit, run `/t deposit` or `/n deposit`. With withdraw, run `/t withdraw` or `/n withdraw`. 
+The safest place for gold is in an ender chest. Gold can also be deposited into the virtual town/nation bank. This will allow residents with appropriate permissions to withdraw gold and use it for town/nation expenses such as claiming. To deposit, run `/t deposit` or `/n deposit`. With withdraw, run `/t withdraw` or `/n withdraw`. 
 
 
 ## How to get gold
 ### Mining
-The beauty of an item based currency is that players can dig it up straight from the ground. Most players prefer to mine around y level 22. Although gold is evenly spread out around the map, some areas are more dug out than others. Avoid Europe and North America if you are a new player starting out. Don't forget to bring your silk touch pick down the mine to double the reward.
+The beauty of an item-based currency is that players can dig it up straight from the ground. Most players prefer to mine around y level 22. Although gold is evenly spread out around the map, some areas are more dugout than others. Avoid Europe and North America if you are a new player starting out.
 
 ### Raiding
 All town protections are removed when a town falls. Find abandoned towns and loot the riches. Stealing is allowed on EarthMC.
 
 ### Voting
-Run the command `/vote` and click the provided links. You will receive a gold crate with a random award up to 64 gold ingots after successfully voting for the server. Keep in mind that you are able to vote without being online, your award will be given to you next time you sign in. The most likely award is 4 gold per vote link. It's also possible to win tridents from voting but they are very rare.
+Run the command `/vote` and click the provided links. You will receive a gold crate with a random award of up to 64 gold ingots after successfully voting for the server. Keep in mind that you are able to vote without being online. Your award will be given to you next time you sign in and can be claimed with `/cr claim`. The most likely award is 4 gold per vote link. It's also possible to win tridents and mending enchantment books from voting, but they are very rare.
```

Comment:
While we're looking at revisions, voting offline was disabled due to spam so that should probably get removed from the page.